src/api/api.ts

```diff
@@ -75,9 +83,16 @@ export class OpenSeaAPI {
 
     switch (config.chain) {
       case Chain.Goerli:
+      case Chain.Sepolia:
```

Comment:
possibly makes sense to add a `is_testnet(chain: Chain)` helper and just use that here, then we can use the helper fn elsewhere in the future as well